sympy/stats/joint_rv_types.py

```diff
@@ -253,3 +253,74 @@ def NormalGamma(syms, mu, lamda, alpha, beta):
     A random symbol
     """
     return multivariate_rv(NormalGammaDistribution, syms, mu, lamda, alpha, beta)
+
+
+#-------------------------------------------------------------------------------
+# Multivariate Beta distribution ---------------------------------------------------------
+
+class MultivariateBetaDistribution(JointDistribution):
+
+    _argnames = ['alpha']
+    is_Continuous = True
+
+    def check(self, alpha):
+        _value_check(len(alpha) >= 2, "At least two categories should be passed.")
+        _value_check(all([a_k > 0 for a_k in alpha]), "Each concentration parameter"
```

Comment:
this will raise an exception if `a_k` is symbolic. `a_k > 0` creates a `GreaterThan` object which cannot be cast to a boolean.

I usually do  `(a_k > 0) == True`.

Reply:
I think that `_value_check` should be able to handle also Relational type conditions. It has `if condition == False`.

Reply:
Maybe ` (a_k > 0) != False` is the correct condition. You would probably allow generic symbols.

Reply:
```python
for a_k in alpha:
    _value_check(a_k > 0, "...")
```

Reply:
@jksuom @Upabjojr I think in `_value_check` the error message should be raised when `condition == None` apart from `condition == False`. As in the following session, `theta.is_positive` is `None` but it will get through checks which require theta to be positive and not negative. 
```
Python 3.6.7 (default, Oct 22 2018, 11:32:17) 
[GCC 8.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from sympy import *
>>> theta = symbols('theta', real = True)
>>> theta.is_positive
>>> theta.is_negative
>>> theta.is_real
True
``` 
PS - I haven't tried the change but may be it can require change at lots of places but I believe that efforts required hold little value when concepts need to be corrected. Though that is just my point of view.

Reply:
If `_value_check` is made to return True if it doesn't raise then this can be written `if all(_value_check(a_k < 0, msg) for a_k in alpha):`